File: java-store-management-system/src/Food.java
import java.time.LocalDate;
import java.util.Date;

public class Food extends Item {
    private Double weight;

    public Food() {
        super();
    }

    public Food(Double weight) {
        super();
        this.weight = weight;
    }

    public Food(String name, Double price, LocalDate expirationDate, Double weight) {
        super(name, price, expirationDate);
        this.weight = weight;
    }

    public Double getWeight() {
        return weight;
    }

    public void setWeight(Double weight) {
        this.weight = weight;
    }

    @Override
    public String toStr() {
        String message = super.toStr();
        message += String.format("\nWeight: %sg", weight);
        return message;
    }
}

File: python-store-management-system/Food.py
from Item import Item


class Food(Item):
    def __init__(self, name, price, expiration_date, weight):
        super().__init__(name, price, expiration_date)
        self.weight = weight

    def __str__(self):
        return f"Item: {self.name}, {self.price}, {self.expiration_date}, {self.weight}"
